docs/admin-guides/managing-catalogs.md

```diff
@@ -49,6 +49,7 @@ Common properties include:
 - warehouse: Warehouse **must be configured**, as it determines where our database and table files should be placed
 - catalog-impl: when the metastore is **Custom**, an additional catalog-impl must be defined, and the user must put the jar package for the custom catalog implementation into the **{ARCTIC_HOME}/lib** directory, **and the service must be restarted to take effect**
 - table.*: If you want to add the same table configuration to all tables under a catalog, you can add `table.` before the configuration key to indicate that it is a table-level configuration. For example, `table.self-optimizing.group`
+- table.filter-regular-expression: If you want to filter the same table under a database, you can add `table.filter-regular-expression` before the configuration key to indicate that it is a table-level configuration. For example, `table.filter-regular-expression`='(A\.a)|(B\.b)'
```

Comment:
You can modify the docs to:

- table.filter-regular-expression: Configure a regular expression to filter tables under databases. The matching will be done in the format of `db.table`. For example, if it is set to '(A\.a)|(B\.b)', it will ignore all tables except for table `a` in database `A` and table `b` in database `B`.